spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -31,8 +31,9 @@
         Subject: API::V2::SerializableSubject,
         PrimarySubject: API::V2::SerializableSubject,
         Provider: API::V2::SerializableProvider,
+        Site: API::V2::SerializableSite,
       },
-      include: %i[subjects sites provider accrediting_provider],
+      include: [:subjects, :sites, :accrediting_provider, :provider, provider: [:sites]],
```

Comment:
Is this actually getting tested?

Reply:
This is the test, if you remove this change the tests will fail.

Reply:
i was kinda expecting a test that was like `returns a course with subject relationships`